cron/centAcl.php

```diff
@@ -98,29 +100,33 @@ function programExit($msg)
                 "INSERT INTO cron_operation (name, system, activate) " .
                 "VALUES ('centAcl.php', '1', '1')"
             );
-            $DBRESULT = $pearDB->query(
+            $dbResult = $pearDB->query(
                 "SELECT id, running FROM cron_operation WHERE name LIKE 'centAcl.php'"
             );
         } catch (\PDOException $e) {
-            print "Error to check if process is running.";
+            print "Error can't check when process is running.";
             exit(1);
         }
-        $data = $DBRESULT->fetch();
-        $appID = $data["id"];
+        $data = $dbResult->fetch();
+        $appId = (int)$data["id"];
         $is_running = 0;
     }
 
     if ($is_running == 0) {
-        $DBRESULT = $pearDB->query(
+        $dbResult = $pearDB->prepare(
             "UPDATE cron_operation SET running = '1', time_launch = '" . time() .
-            "' WHERE id = '$appID'"
+            "' WHERE id = :appId"
         );
+        $dbResult->bindValue(':appId', $appId, \PDO::PARAM_INT);
+        $dbResult->execute();
     } else {
         if ($nbProc <= 1) {
             $errorMessage = "According to DB another instance of centAcl.php is already running and I found " .
                 $nbProc . " process...\n";
-            $errorMessage .= "Executing query: UPDATE cron_operation SET running = 0 WHERE id =  '$appID'";
-            $pearDB->query("UPDATE cron_operation SET running = '0' WHERE id = '$appID'");
+            $errorMessage .= "Executing query: UPDATE cron_operation SET running = 0 WHERE id =  ' . $appId . '";
```

Comment:
it will display dots in error message cause it's encapsulated between double quotes